springboot/demo/src/main/java/com/example/demo/controller/HelloController.java

```diff
@@ -0,0 +1,14 @@
+package com.example.demo.controller;
+
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+public class HelloController {
+
+    @GetMapping("/")
```

Comment:
上記指摘コメントにともない、修正してください。
```suggestion
    @GetMapping("/helloworld")
```